test/Microsoft.AspNet.Razor.Test/Parser/CSharp/CSharpDirectivesTest.cs

```diff
@@ -12,29 +12,16 @@ namespace Microsoft.AspNet.Razor.Test.Parser.CSharp
     public class CSharpDirectivesTest : CsHtmlCodeParserTestBase
     {
         [Fact]
-        public void TagHelperPrefixDirective_NoValueSucceeds()
-        {
-            ParseBlockTest("@tagHelperPrefix \"\"",
-                new DirectiveBlock(
-                    Factory.CodeTransition(),
-                    Factory
-                        .MetaCode(SyntaxConstants.CSharp.TagHelperPrefixKeyword + " ")
-                        .Accepts(AcceptedCharacters.None),
-                    Factory.Code("\"\"")
-                        .AsTagHelperPrefixDirective(string.Empty)));
-        }
-
-        [Fact]
         public void TagHelperPrefixDirective_Succeeds()
         {
-            ParseBlockTest("@tagHelperPrefix \"Foo\"",
```

Comment:
Nope, have not preserved any tests that have quotes. Don't typically maintain tests for old/changed features.

As for what the user will see if they still have quotes. You'll get a design time error because the code generation will look like: `_syntaxhelper = ""value""`